feat: add useDeepCompareEffect (#1042)
alibaba/hooks: packages/hooks/src/useDeepCompareEffect/index.en-US.md

---
title: useDeepCompareEffect
nav:
  title: Hooks
  path: /hooks
group:
  title: SideEffect
  path: /side-effect
---

# useDeepCompareEffect

> Usage is the same as useEffect, but deps use [lodash.isEqual](https://lodash.com/docs/4.17.15#isEqual) to compare.

Reviewer: ```suggestion
Usage is the same as `useEffect`, but deps are compared with [lodash.isEqual](https://lodash.com/docs/4.17.15#isEqual).
```
